client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -24,33 +25,64 @@ const DomainRegistrationThankYouProps = ( {
 			selectedSiteSlug,
 		},
 		'REGISTRATION',
-		false
+		true
 	);
 
 	const returnProps: DomainThankYouProps = {
+		thankYouNotice: {
+			noticeTitle: translate(
+				'It may take up to 30 minutes for your domain to start working properly.'
+			),
+			noticeIconCustom: <Icon icon={ info } size={ 24 } />,
+		},
 		sections: [
 			{
 				sectionKey: 'domain_registration_whats_next',
 				sectionTitle: translate( 'What’s next?' ),
 				nextSteps: [
+					...( professionalEmail ? [ professionalEmail ] : [] ),
+					...( ! selectedSiteSlug
+						? [
+								{
+									stepKey: 'domain_registration_whats_next_create-site',
+									stepTitle: translate( 'Add a site' ),
+									stepDescription: translate( 'Choose a theme, customize and launch your site.' ),
+									stepCta: (
+										<FullWidthButton
+											href={ createSiteFromDomainOnly( domain, null ) }
+											busy={ false }
+											disabled={ false }
+										>
+											{ translate( 'Create a site' ) }
+										</FullWidthButton>
+									),
+								},
```

Comment:
This works, but I think it would be more readable to extract at least this object to a variable 🙂